src/components/AsideBlock/AsideBlock.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+
+import './AsideBlock.css';
```

Comment:
All styles here should be written in a scss file. Then it is compiled into css, which is imported by a component. To inject a new css stylesheet first write this new scss file, then this project was launched by create-react-app, and they have created the build process so that you should then do 
`npm run start`
again so that new css file could be generated based on that scss file.

Note that css files are not saved to git and ignored in commits. Only scss files are saved to git.

Please, convert css into scss as i do not see any stylesheets attached to the component.